### PR TITLE
Implement API: GET /wallets/{id} (get wallet info by name or id)

### LINKED ISSUE #21: Implement API: GET /wallets/{id} (get wallet info by name or id)
https://github.com/Greenstand/treetracker-query-api/blob/5b796230237325ee526616d5d3d9d1281fbe8b92/docs/api/spec/query-api.yaml#L537-L548

To implement an endpoint:

* Please write an e2e test to cover it.
* Please follow the guide in the 'readme' to follow our architecture on the server-side.

---

Some hints:

- Please read our readme for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

### PR DESCRIPTION
**Pull request by pierrelstan**
The following changes have been implemented:

- Create  wallets model 
-  Create wallets test model
-  Create wallets e2e test   
- Create wallets router 
- Create interface wallets
-  Create wallets repository
- Update README.md (end-to-end-test)

resolves #21

### CODE REVIEW

**Comment by dadiorchen** [comment]
@pierrelstan did the e2e test pass?

**Comment by pierrelstan** [comment]
> @pierrelstan did the e2e test pass?

@dadiorchen  locally it returns 
`{
"code": 500,
"message": "Unknown error (self signed certificate in certificate chain)"
}`

**Comment by pierrelstan** [comment]
@dadiorchen  when I tried to query the wallet or wallets table :
`SELECT *  from wallet`  I got an error  ` SELECT *
          FROM wallet
           - relation "wallet" does not exist`
Does that mean I need to create a table's wallet before being able to get by id or name from the wallet?
or does the data for the wallet is from another table name?

**Comment by dadiorchen** [comment]
use `wallet.wallet` instead, the former is the scheme name, we put wallet/token tables into the `wallet` schema in the postgresql

**Comment by pierrelstan** [comment]
> use `wallet.wallet` instead, the former is the scheme name, we put wallet/token tables into the `wallet` schema in the PostgreSQL


Thank you.

File: README.md [hunk 1]
```diff
@@ -109,4 +109,22 @@ Run tests:
...
+`
+{
+  "code": 500,"message": "Unknown error (self signed certificate in certificate chain)"
+}
+`
+
+Run tests:
+
+``
+DATABASE_URL=postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/treetracker?ssl=true NODE_TLS_REJECT_UNAUTHORIZED='0'  npm run test-e2e
```
Comment (dadiorchen) [inline README.md]: @pierrelstan you need to hide the real database string, with some placeholder words/dummy string.

Comment (dadiorchen) [reply README.md]: Otherwise the DB is easy to be hacked

**Review by dadiorchen — CHANGES_REQUESTED** [review]

**Comment by dadiorchen** [comment]
@pierrelstan thank you, great job!

**Pull request merged**